src/sentry/api/serializers/models/user.py

```diff
@@ -150,10 +152,18 @@ def get_attrs(self, item_list, user):
             user__in=item_list,
         ), 'user_id')
 
+        memberships = manytoone_to_dict(OrganizationMember.objects.filter(
+            user__in=item_list,
+            organization__status=OrganizationStatus.VISIBLE,
+        ), 'user_id')
+
         for item in item_list:
             attrs[item]['authenticators'] = authenticators[item.id]
             attrs[item]['permissions'] = permissions[item.id]
 
+            # org can reset 2FA if the user is only in one org
+            attrs[item]['canReset2fa'] = len(memberships[item.id]) == 1
```

Comment:
Hm I went with no, the user shouldn't be able to reset their 2FA from this page. I think users can only read the member details page, but it takes manager and up permissions to take any action. Thought it would be more consistent to keep this page for managing members, since users can remove their 2FA in user settings. What do you think?

Reply:
That sounds good to me.